syntax = "proto3";

package fixbug;
option cc_generic_services = true;

message ResultCode {
    int32 errcode = 1;
    bytes errmsg = 2;
}

message LoginReq {
    bytes name = 1;
    bytes pwd  = 2;
}

message LoginResp {
    ResultCode result = 1;
    bool succes = 2;
}

message RegisterReq {
    uint32 id = 1;
    bytes name = 2;
    bytes pwd = 3;
}

message RegisterResp {
    ResultCode result = 1;
    bool success = 2;
}

service UserServiceRpc {
    rpc login(LoginReq) returns (LoginResp);
    rpc reg(RegisterReq) returns (RegisterResp);
}